src/customizations/components/theme/Navigation/Navigation.jsx

```diff
@@ -14,6 +14,7 @@ import { connect } from 'react-redux';
 import { NavLink } from 'react-router-dom';
 import { compose } from 'redux';
 import { Button, Icon, Image, Menu } from 'semantic-ui-react';
+import OutsideClickHandler from 'react-outside-click-handler';
```

Comment:
@dobri1408 could you please consider using https://github.com/eea/volto-accordion-block/blob/a240e31c3cdebea22f82b9b3f8308119cfe6d8a2/src/components/manage/Blocks/Accordion/EditBlockWrapper.jsx#L9
for detecting click inside a DOM node, this also avoids introduction of a new library.